src/Controls/tests/Core.UnitTests/KeyboardAcceleratorTypeConverterUnitTests.cs

```diff
@@ -0,0 +1,15 @@
+using Xunit;
+
+namespace Microsoft.Maui.Controls.Core.UnitTests
+{
+	public class KeyboardAcceleratorTypeConverterUnitTests : BaseTestFixture
+	{
+		[Fact]
+		public void TestKeyboardAcceleratorTypeConverter()
```

Comment:
Ah, I see this was renamed actually...

Reply:
well I can get rid of this file and keep the old one?